app/models/well.rb

```diff
@@ -220,13 +220,8 @@ def qc_results_by_key
   end
 
   def qc_result_for(key)
-    result =
-      if key == 'quantity_in_nano_grams'
-        well_attribute.quantity_in_nano_grams
-      else
-        results = qc_results_by_key[key]
-        results.first.value if results.present?
-      end
+    results = qc_results_by_key[key]
+    result = results.first.value if results.present?
```

Comment:
`quantity_in_nanograms` was returning the computed value `estimated_volume * concentration` after checking nil and negative values. Now the change is returning the value directly. Is this correct?